### PR TITLE
[fix](information_schema)fix bug that metadata_name_ids error tableid and append information_schema case. 

### PR DESCRIPTION
**Pull request by hubgeter**
fix bug that  #24059 .
Added some information_schema scanner tests.
files
schema_privileges
table_privileges
partitions
rowsets
statistics
table_constraints

Based on infodb_support_ext_catalog=false, it currently includes tests for all tables under the information_schema database.

## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by hubgeter** [comment]
run buildall

**Comment by hubgeter** [comment]
run buildall

File: regression-test/suites/query_p0/system/test_query_sys_tables.groovy [hunk 1]
```diff
@@ -87,12 +87,53 @@ suite("test_query_sys_tables", "query,p0") {
...
+            c string ) 
+        DISTRIBUTED BY HASH(`a`) BUCKETS 1 
+        PROPERTIES (
+            "replication_num" = "1",
+            "disable_auto_compaction" = "true",
+            "enable_single_replica_compaction"="true"
+        );
+    """
+    
+    List<List<Object>> rowsets_table_name_tablts   = sql """ show tablets from ${rowsets_table_name} """
```
[the remark this answers is not in the document]
Comment (hubgeter) [reply regression-test/suites/query_p0/system/test_query_sys_tables.groovy]: Only the two tables information_schema.statistics and information_schema.table_constraints have no test case.

**Review by morningman — DISMISSED** [review]
LGTM

File: regression-test/suites/query_p0/system/test_query_sys_tables.groovy [hunk 2]
```diff
@@ -87,12 +87,53 @@ suite("test_query_sys_tables", "query,p0") {
...
+    // have no impl(fe:listTablePrivilegeStatus)
+    qt_desc_table_privileges """desc `information_schema`.`table_privileges` """
+    qt_select_schema_privileges """  select * from information_schema.table_privileges   """  
+
 
     // test partitions
     // have no impl
+    qt_desc_partitions """ desc `information_schema`.`partitions` """ 
+    qt_select_partitions """ select * from  `information_schema`.`partitions`; """ 
+
```
Comment (BePPPower) [inline regression-test/suites/query_p0/system/test_query_sys_tables.groovy:108]: we'd better use `order_qt` for select statement, rather than `qt`.

**Comment by hubgeter** [comment]
run buildall

**Review by morningman — APPROVED** [review]
LGTM

**Review by BePPPower — APPROVED** [review]
LGTM

**Pull request merged**